Competition2017/src/competition/subsystems/shooter_belt/ShooterBeltSubsystem.java

```diff
@@ -56,4 +64,17 @@ public void intakeUsingSpeed() {
     public void stop(){
         setPower(0);
     }
+    
+    @Override
+    public void updatePeriodicData() {
+        super.updatePeriodicData();
+        
+        Point primaryMotorPoint = Point.measurement(this.getClass().getSimpleName())
```

Comment:
You do this so often, since many of our subsystems have motors. Maybe the XCANTalon should expose a `getTelemetryPoint(String callerName, String tag)` or something like that?